app/controllers/Application.scala

```diff
@@ -1,17 +1,22 @@
 package controllers
 
+import com.typesafe.config.Config
+import javax.inject.Inject
 import play.api.mvc._
-import play.api.Play
 
-object Application extends Controller {
+class Application @Inject()(cc: ControllerComponents, underlyingConfig: Config) extends AbstractController(cc) {
+  private var _documentation: Seq[(String, String, String)] = Seq()
 
-  def index = Action {
-    import play.api.Play.current
-    Ok(views.html.index(Play.routes.documentation))
+  def documentation: Seq[(String, String, String)] = _documentation
+  def documentation_= (newValue: Seq[(String, String, String)]): Unit = {
+    _documentation = newValue
   }
```

Comment:
Are we mutating here on purpose?

Reply:
This was my way of getting round the circular dependency between the router and the application controller. You'll see at the end of the `PrismComponents` class in `PrismApplicationLoader` that after instantiating the application controller and router we call `homeController.documentation = router.documentation`. 

I'm sure there's a nicer way of doing this so I'm all ears 👂

Reply:
🛠️